pkg/controller/istiocontrolplane/pruningdetails.go

```diff
@@ -70,62 +70,62 @@ var (
 	}
 
 	// ordered by which types should be deleted, first to last
```

Comment:
also update the comment of how to use the boolean? like `false` means xxx?

Reply:
done